src/backend/base/langflow/api/v1/folders.py

```diff
@@ -121,19 +122,25 @@ def read_folder(
         raise HTTPException(status_code=404, detail="Folder not found")
 
     try:
-        stmt = select(Flow).where(Flow.folder_id == folder_id, Flow.user_id == current_user.id)
-
-        if Flow.updated_at is not None:
-            stmt = stmt.order_by(Flow.updated_at.desc())  # type: ignore[attr-defined]
-        if is_component:
-            stmt = stmt.where(Flow.is_component == True)  # noqa: E712
-        if is_flow:
-            stmt = stmt.where(Flow.is_component == False)  # noqa: E712
-        if search:
-            stmt = stmt.where(Flow.name.like(f"%{search}%"))  # type: ignore[attr-defined]
-        paginated_flows = paginate(session, stmt, params=params)
-
-        return FolderWithPaginatedFlows(folder=FolderRead.model_validate(folder), flows=paginated_flows)
+        if params and params.page and params.size:
+            stmt = select(Flow).where(Flow.folder_id == folder_id)
+
+            if Flow.updated_at is not None:
+                stmt = stmt.order_by(Flow.updated_at.desc())  # type: ignore[attr-defined]
+            if is_component:
+                stmt = stmt.where(Flow.is_component == True)  # noqa: E712
+            if is_flow:
+                stmt = stmt.where(Flow.is_component == False)  # noqa: E712
+            if search:
+                stmt = stmt.where(Flow.name.like(f"%{search}%"))  # type: ignore[attr-defined]
+            paginated_flows = paginate(session, stmt, params=params)
+
+            return FolderWithPaginatedFlows(folder=FolderRead.model_validate(folder), flows=paginated_flows)
+
+        flows_from_current_user_in_folder = [flow for flow in folder.flows if flow.user_id == current_user.id]
+        folder.flows = flows_from_current_user_in_folder
+        return folder  # noqa: TRY300
```

Comment:
You could fix this if you want :shrug: https://docs.astral.sh/ruff/rules/try-consider-else/

Reply:
this is a little tricky
the format is formatting the code and the ruff is complaining of the formatted code